keps/sig-scheduling/5234-dra-resourceslice-mixins/README.md

```diff
@@ -0,0 +1,912 @@
+
+###### Will enabling / using this feature result in any new API calls?
+
+No
+
+###### Will enabling / using this feature result in introducing new API types?
+
+No
+
+###### Will enabling / using this feature result in any new calls to the cloud provider?
+
+No
+
+###### Will enabling / using this feature result in increasing size or count of the existing API objects?
+
+Yes and no. It does add additional fields, which increases the worst case
+size of the ResourceSlice object. However, it also provides features that
+allows drivers to represent devices and counter sets in a more compact way,
+thereby potentially reducing the size of the ResourceSlice object.
+
+###### Will enabling / using this feature result in increasing time taken by any operations covered by existing SLIs/SLOs?
+
+Flattening the devices and counter sets will require slightly more work, but
+this is unlikely to have any meaningful impact on the time used for allocation.
+
+It does allow DRA driver authors to create more complex devices, with a larger
+number of counters. It also allows for larger number of counters in the counter
+sets. This can worsen the worst-case scheduling performance.
+
+###### Will enabling / using this feature result in non-negligible increase of resource usage (CPU, RAM, disk, IO, ...) in any components?
+
+No
+
+###### Can enabling / using this feature result in resource exhaustion of some node resources (PIDs, sockets, inodes, etc.)?
+
+No, because the feature is not used on nodes.
+
+### Troubleshooting
+
+<!--
+This section must be completed when targeting beta to a release.
+
+For GA, this section is required: approvers should be able to confirm the
+previous answers based on experience in the field.
+
+The Troubleshooting section currently serves the `Playbook` role. We may consider
+splitting it into a dedicated `Playbook` document (potentially with some monitoring
+details). For now, we leave it here.
+-->
+
+###### How does this feature react if the API server and/or etcd is unavailable?
+
+###### What are other known failure modes?
+
+<!--
+For each of them, fill in the following information by copying the below template:
+  - [Failure mode brief description]
+    - Detection: How can it be detected via metrics? Stated another way:
+      how can an operator troubleshoot without logging into a master or worker node?
+    - Mitigations: What can be done to stop the bleeding, especially for already
+      running user workloads?
+    - Diagnostics: What are the useful log messages and their required logging
+      levels that could help debug the issue?
+      Not required until feature graduated to beta.
+    - Testing: Are there any tests for failure mode? If not, describe why.
+-->
+
+###### What steps should be taken if SLOs are not being met to determine the problem?
+
+## Implementation History
+
+- 1.33: first KEP revision as part of the Partitionable Devices KEP
+- 1.34: split out into a separate KEP and initial implementation.
```

Comment:
This 1.34 entry isn't "history" yet. If we merge this now and then don't follow through in 1.34, we'll have an incorrect statement in the KEP. Perhaps add `(tentative)`?

Reply:
If we're being picky, the splitting is history (with this merge). But the initial implementation is not, so maybe we just leave that bit off.

Reply:
Yeah, I have removed the implementation bit.